api/src/main/java/org/openmrs/module/fhir2/model/FhirTaskOutput.java

```diff
@@ -25,7 +25,7 @@
  */
 @Data
 @NoArgsConstructor
-@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
```

Comment:
```suggestion
```

Reply:
@ibacher , removing `@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)` doesnt solve the issue. for some reason it still fails.
setting `callSuper = true ` solves it.